addons/account/models/account_move_line.py

```diff
@@ -415,6 +415,12 @@ class AccountMoveLine(models.Model):
              "associated partner",
     )
     is_refund = fields.Boolean(compute='_compute_is_refund')
+    # === Check Info === #
+    check_number = fields.Char(
+        string="Check Number",
+        store=False,
+        compute='_compute_check_number',
```

Comment:
It is the default value for computed fields
```suggestion
        compute='_compute_check_number',
```